### PR TITLE
docs: Add more context for jobs

### PR DESCRIPTION
**Pull request by eseidel**
"The Flutter Company" is a mash of multiple sections and needs to be trimmed down further.

### CODE REVIEW

**Comment by eseidel** [comment]
I've been fiddling with this for a while and just need to finish it.

File: src/pages/jobs.md [hunk 1]
```diff
@@ -216,12 +264,13 @@ and serve an influx of customers in capacities other than writing software.
...
-career/senior (3-5 years experience) role, involving involving nearly unbounded
-growth potential. This role will be salary/equity compensated, not commission.
+at minimum an early-career role (3+ years experience), involving involving
+nearly unbounded growth potential. More experience applicants are welcome, there
+is plenty to do.  This role will be salary/equity compensated, not commission.
 Our incentives should be aligned to build/grow the business over a long time
 horizon.
 
-130-175k USD salary, 1.5% equity, and
+170-220k USD salary, 1.5% equity, and
```
Comment (eseidel) [inline src/pages/jobs.md:273]: I also need to check these numbers.  I haven't looked at the market. I had planned to do that last week but then got sick.

File: src/pages/jobs.md [hunk 2]
```diff
@@ -15,26 +15,94 @@ We're hiring
...
-## Why should I work at Shorebird?
+Shorebird is a small, venture-funded startup (est Jan 2023) trying to make
+Flutter the default way to write apps. So far we've built and shipped Code Push
+for Flutter to let businesses over the air update their apps instantly on users
+phones.
+
+## Why work at a startup?
+
+Having worked at both, I can tell you, startups feel very different from big
+companies and big teams.
```
Comment (bryanoltman) [inline src/pages/jobs.md:26]:
```suggestion
Having worked at both large and small companies, I can tell you, startups feel very different from big
companies and big teams.
```

File: src/pages/jobs.md [hunk 3]
```diff
@@ -15,26 +15,94 @@ We're hiring
...
+
+Working at a startup means:
+
+- Shipping. We ship 5 times a day, not 5 times a year.
```
Comment (bryanoltman) [inline src/pages/jobs.md:30]: 5 times a day is maybe a bit of a stretch :)

File: src/pages/jobs.md [hunk 4]
```diff
@@ -15,26 +15,94 @@ We're hiring
...
+- (Growth)[https://www.paulgraham.com/growth.html]. We are trying to
+  grow quickly (revenues, customers, products, people, etc). Growth means
+  opportunity, variety and challenge.
+- Ambition. Hard work is needed and rewarded. Your colleagues are highly
+  engaged, want to be here, want to work hard, and want to learn with and from
+  you.
+- Big upside. We pay well relative to other startups, but less (cash) than Big
+  Tech. Instead we offer lots of equity. Equity means you own a percentage of
+  Shorebird, and have a significant financial share in the company's success.
```
Comment (bryanoltman) [inline src/pages/jobs.md:39]:
```suggestion
  Shorebird, and have a significant financial stake in the company's success.
```

File: src/pages/jobs.md [hunk 5]
```diff
@@ -15,26 +15,94 @@ We're hiring
...
+- Ownership. Team is very small. For anything to be done, that means one of
```
Comment (bryanoltman) [inline src/pages/jobs.md:40]:
```suggestion
- Ownership. The team is very small. For anything to be done, that means one of
```

File: src/pages/jobs.md [hunk 6]
```diff
@@ -15,26 +15,94 @@ We're hiring
...
+- No layers. Direct access to customers. Direct access to management (the CEO
+  is the only manager). Nothing between you and direct action.
+- Few distractions. Engineers don't have meetings. There are no perf reviews. We
+  show up and do.
+
+## Problem: App Development is Broken
+
+App development is broken. The default answer to "write an app" is write it once
+for iOS, once for Android and once for the web (possibly more places).
+Businesses write the same app at least twice at 2x the necessary costs, with
```
Comment (bryanoltman) [inline src/pages/jobs.md:53]:
```suggestion
App development is broken. The default answer to "write an app" is write it once
for iOS, once for Android, and once for the web (possibly more places).
Businesses write the same app at least twice at 2x the necessary costs, with
```

File: src/pages/jobs.md [hunk 7]
```diff
@@ -15,26 +15,94 @@ We're hiring
...
+I (and many others) tried to make the Web (PWAs, etc) solve "write once; run
+anywhere" a decade ago and failed. Hence we built Flutter to let Google and
+everyone write high-quality app experiences and have them go everywhere.
+
+Flutter is now the most popular solution for writing multi-platform apps. Over
+a million developers use Flutter every month, and growing. Flutter has proven
+that high-quality multi-platform is possible.
+
+However, still most apps don't use Flutter. And even some that do, sometimes
+struggle. Adopting Flutter can be challenging since the default offerings from
```
Comment (bryanoltman) [inline src/pages/jobs.md:71]: > And even some that do, sometimes struggle

I'm not 100% sure what this is trying to say

File: src/pages/jobs.md [hunk 8]
```diff
@@ -15,26 +15,94 @@ We're hiring
...
+Google.
+
+The Flutter project itself is also sometimes a funny fit inside Google (the home
+of Android, Ads and Chrome) and long term would benefit from an economic entity
+wholly aligned with Flutter's goals (not just trying to use it to sell Ads or
+Cloud).
+
+All these problems are why after taking Flutter as far as I could within
+Google, I left to build a company around Flutter: Shorebird.
+
```
Comment (bryanoltman) [inline src/pages/jobs.md:82]:
```suggestion
All these problems are why, after taking Flutter as far as I could within
Google, I left to build a company around Flutter: Shorebird.

```

**Review by bryanoltman — APPROVED** [review]

File: src/pages/jobs.md (hunk 3 above)
Comment (eseidel) [reply src/pages/jobs.md:30]: True, I can change it to 5 times a week. Although sometimes we ship 3 times a day 🤣

**Pull request merged**